src/css/workbench-custom.css

```diff
@@ -409,3 +409,46 @@
 .help-info b {
     font-weight: 500;
 }
+
+.dropdown-menu.multi-select {
```

Comment:
Do you mind if we scope all selectors below under this which is the component root?

Reply:
done